0000-payment-pointers/0000-payment-pointers.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Payment Pointers
+draft: 1
+---
+# Payment Pointers and Payment Setup Protocols
+
+## Abstract
+
+Payment pointers are a standardized identifier for accounts that are able to recieve payments. In the same way that an email address provides an identifier for a mailbox in the email ecosystem a payment pointer is used by an account holder to share the details of their account with anyone that wishes to make a payment to them.
+
+A payment pointer can be normalized to an "https" URL that provides the location of a payment setup service endpoint at which a sender can initiate a payment to the receiver.
```

Comment:
I think "normalize" means change from one of the variations to the canonical one, but the https URL is not itself a payment pointer, so I would rather use the verb "dereference" instead.

Reply:
Used resolved as that's what I have used consistently